apps/frontend/src/app/2_molecules/AmountRenderer/AmountRenderer.tsx

```diff
@@ -116,18 +122,16 @@ export const AmountRenderer: FC<AmountRendererProps> = ({
       dataAttribute={dataAttribute}
     >
       {isAnimated ? (
-        <div>
-          <CountUp
-            start={0}
-            end={formattedValue}
-            duration={0.7}
-            separator={thousand}
-            decimals={formattedValueDecimals}
-            decimal={decimal}
-            prefix={shouldShowRoundingPrefix ? '~ ' : ''}
-            suffix={` ${suffix.toUpperCase()}`}
-          />
-        </div>
+        <CountUp
```

Comment:
Tooltip doesn't work without the `div` you deleted.

Reply:
Got it, but still will wrap both of children to a span instead of div (previous version made animated renderer block element while normal one was inline)